datajunction-server/datajunction_server/api/sql.py

```diff
@@ -35,6 +36,13 @@ async def get_measures_sql_for_cube(
     dimensions: List[str] = Query([]),
     filters: List[str] = Query([]),
     *,
+    include_all_columns: bool = Query(
```

Comment:
Something sneaked in the PR :)

Reply:
Oops yeah... I also needed to do this and forgot that I was making changes at the same time 😅